code/modules/mob/living/carbon/xenomorph/xeno_defines.dm

```diff
@@ -286,6 +286,7 @@
 	var/evolution_without_ovipositor = TRUE //Temporary for the roundstart.
 	var/allow_queen_evolve = TRUE // Set to true if you want to prevent evolutions into Queens
 	var/hardcore = FALSE // Set to true if you want to prevent bursts and spawns of new xenos. Will also prevent healing if the queen no longer exists
+	var/no_burrowed = FALSE  // Can have burrowed from latejoin or not
```

Comment:
cpdedpc

Reply:
Uh. Not sure what you mean.

Reply:
See https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/AUTODOC.md#documenting-a-variabledefine

Reply:
For example something like this:
```
	/// Set to true if you want to prevent bursts and spawns of new xenos. Will also prevent healing if the queen no longer exists
	var/hardcore = FALSE // Set to true if you want to prevent bursts and spawns of new xenos. Will also prevent healing if the queen no longer exists.
	/// Set to false if you want to prevent burrowed larva increasing from latejoin
	var/allow_late_join_larva = TRUE
```
To address the codedoc request and the negative in the name (you'll have to reverse all the other usage too)

Reply:
All right. Sorry for being slow. Will do that

Reply:
Now it changed as morrow wanted, and what difference.... between // and ///?

Reply:
The difference is that /// is an autodoc - see details in that link earlier here.

Reply:
Oh. My bad.